docs/application/vstools/tools/unit-test-with-code-coverage.md

```diff
@@ -0,0 +1,120 @@
+      
+      <img src="./media/utc_1.png" alt="Creating gtest project part: 1" width="980"/>
+      
+      <!-- ![Test results](./media/utc_1.png) -->
+
+      <img src="./media/utc_2.png" alt="Creating gtest project part: 2" width="980"/>
+    
+      <!-- ![Test results](./media/utc_2.png) -->
+    
+  2. In the **Configure your new project** window, select a name for the **Unit Test** project > click on **create** button > select **gtest** template > press **OK** as illustrated in the following figure:
+      
+      <img src="./media/utc_3.png" alt="Creating gtest project part: 3" width="980"/>
+      
+      <!-- ![Test results](./media/utc_3.png) -->
+
+## To configure the Unit Test project for Native App
+1. Right-click on the **Unit Test** project > select **Add Tizen Project Dependency** > Select the **Native App** > press **OK** as illustrated in the following figure:
+
+    <img src="./media/utc_4.png" alt="Add tizen dependency" width="480"/>
+
+    <!-- ![Test results](./media/utc_4.png) -->
+
+2. Again right-click on the **Unit Test** project > select **Set as Startup Project**
+
+3. To use the test project, follow these steps:
+
+    * `<TEST_PROJECT_HOME>/<TEST_PROJECT_NAME>/src/<TEST_PROJECT_NAME>_test.cpp` file.
+
+    * Add a `TEST_F()` test case.
+    
+      Each `TEST_F()` test case is independent. If the `TEST_F()` test case is associated with a fixture class name, the test case runs based on that fixture class.
+    
+    * Add assertions:
+
+        <img src="./media/utc_5.png" alt="Adding assertion" width="980"/>
+
+        <!-- ![Test results](./media/utc_5.png) -->
+
+      The Unit Test tool supports basic assertions, binary comparison, and string comparison in the **gtest**. For more information, check [Google Test Advanced Guide](https://github.com/google/googletest/blob/main/docs/advanced.md).
+
+    * Call a method of the main Native project in the test case:
+
+        <img src="./media/utc_9.png" alt="Calling ADD method" width="480"/>
+
+        <!-- ![Test results](./media/utc_9.png) -->
+
+      The header of the Native App should be included in the Unit Test project cpp file (in this example: the `ADD()` method is described in `tizennative3.c`, so the header `tizennative3.h` is included). As illustrated in the following figure:
+      
+      In case if the Native project is written in C code, use extern to include the header (To know more details about extern, visit [extern (C++)](https://learn.microsoft.com/en-us/cpp/cpp/extern-cpp?view=msvc-170)). 
+      
+        <img src="./media/utc_10.png" alt="adding extern for c" width="480"/>
+
+        <!-- ![Test results](./media/utc_10.png) -->
+
+      The method declaration should be mentioned in the header file.
+
+        <img src="./media/utc_8.png" alt="method mention in header file" width="480"/>
+
+        <!-- ![Test results](./media/utc_8.png) -->
+
+    * Enclose the main() function of the Native project with `TEST_BUILD`:
+      ```cpp
```

Comment:
The code section is highlighted in the github preview but not in the website. C++ highlighter is supposed to work but not sure why it is not working here. Please try replacing line 86 by:
```csharp